pom.xml

```diff
@@ -478,6 +471,7 @@
         <module>integration/spark2</module>
         <module>integration/hive</module>
         <module>integration/presto</module>
+        <module>streaming</module>
```

Comment:
please add streaming module to "build-all" profile

Reply:
fixed